<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="010223HP840G528" failures="1" tests="2" name="com.ep.cucumber.runner.RunnerClass1" time="4.499" errors="0" timestamp="2023-08-30T19:09:52 IST" skipped="0">
  <testcase classname="com.ep.cucumber.runner.RunnerClass1" name="runScenario" time="1.280"/>
  <system-out/>
  <testcase classname="com.ep.cucumber.runner.RunnerClass1" name="runScenario" time="3.219">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.ep.api.stepdefinition.PetStoreUserSteps.the_status_code_should_be(PetStoreUserSteps.java:66)
at âœ½.The status code should be "201"(file:///D:/APIFramework/SampleApi/src/test/resources/features/PetStoreUser.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- com.ep.cucumber.runner.RunnerClass1 -->
